packages/forms/src/Components/CodeEditor.php

```diff
@@ -12,4 +13,18 @@ class CodeEditor extends Field
      * @var view-string
      */
     protected string $view = 'filament-forms::components.code-editor';
+
+    protected Language $language = Language::PlainText;
+
+    public function language(Language $language): static
+    {
+        $this->language = $language;
+
+        return $this;
+    }
+
+    public function getLanguage(): Language
+    {
+        return $this->language;
+    }
```

Comment:
Please allow closure functions to be passed and evaluated

Ensure that the `<UtilityInjection>` component is used in the docs to reflect this